components/map/index.js

```diff
@@ -102,14 +103,31 @@ export default class TheMap extends Component {
               })
             );
           })}
-          <Marker
-            coordinate={{
-              latitude: this.props.updatedRegion.latitude,
-              longitude: this.props.updatedRegion.longitude
-            }}
-            title="title"
-            description="description"
-          />
+
+          {
+            this.props.nearbyMarkers.length > 0
```

Comment:
This ternary operation is very hard to read at first glance. Perhaps auxiliary functions for the if and else conditions can improve that?